test/spec/a.rb

```diff
@@ -0,0 +1,16 @@
+it 'should search by keywords', bin4: true, quick: true do
```

Comment:
Why the name of this file is `a.rb` and it contains one single test out of a spec block?